qa/integration-tests/src/test/java/io/zeebe/broker/it/gateway/GatewayHealthIndicatorsIntegrationTest.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.broker.it.gateway;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.data.Offset.offset;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.zeebe.broker.it.gateway.GatewayHealthIndicatorsIntegrationTest.Config;
+import io.zeebe.gateway.Gateway.Status;
+import io.zeebe.gateway.impl.SpringGatewayBridge;
+import io.zeebe.gateway.impl.broker.cluster.BrokerClusterState;
+import io.zeebe.gateway.impl.probes.health.ClusterAwarenessHealthIndicator;
+import io.zeebe.gateway.impl.probes.health.PartitionLeaderAwarenessHealthIndicator;
+import io.zeebe.gateway.impl.probes.health.StartedHealthIndicator;
+import io.zeebe.util.health.MemoryHealthIndicator;
+import java.util.List;
+import java.util.function.Supplier;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = Config.class)
+public class GatewayHealthIndicatorsIntegrationTest {
```

Comment:
I would expect in the integration test that we have the standalone gateway and broker running and test the health/liveness checks?

Reply:
It feels like this should be moved to the broker/gateways tests? And here we should relay tests that this returns up and healthy etc if you requests the url and everything is started.